src/NuGet.Core/NuGet.Packaging/Signing/Utility/CertificateChainUtility.cs

```diff
@@ -6,6 +6,7 @@
 using System.Linq;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
+using System.Text;
 using NuGet.Common;
```

Comment:
I don't know why this file was even changed. All these changes can probably be undone.

Reply:
Thanks! Changed.